### ISSUE TITLE
Error while trying to port it to Docker container based out of Linux

### ISSUE THREAD

**Issue by abinjaik** [dotnet/winforms-datavisualization#39]
I successfully moved my charts to .NET core, but when I am trying to containerize its giving the error on UseWindowsForms flag. Any insights?

#16 0.751   Determining projects to restore...
#16 1.020   All projects are up-to-date for restore.
#16 1.1**95 /usr/share/dotnet/sdk/3.1.404/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.DefaultItems.targets(349,5): warning NETSDK1107: Microsoft.NET.Sdk.WindowsDesktop is required to build Windows desktop applications. 'UseWpf' and 'UseWindowsForms' are not supported by the current SDK. [/src/ChartLib/ChartLib.csproj]**
#16 1.879 ChartCore.cs(15,19): error CS0012: The type 'Control' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'. [/src/ChartLib/ChartLib.csproj]
#16 1.879 ChartCore.cs(15,19): error CS1061: 'Chart' does not contain a definition for 'Height' and no accessible extension method 'Height' accepting a first argument of type 'Chart' could be found (are you missing a using directive or an assembly reference?) [/src/ChartLib/ChartLib.csproj]
#16 1.879 ChartCore.cs(16,19): error CS0012: The type 'Control' is defined in an assembly that is not referenced. You must add a reference to assembly 'System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'. [/src/ChartLib/ChartLib.csproj]

**Comment by RussKie** [comment]
The project file needs to have the following by which it reference Windows Forms runtime.
```xml
  <PropertyGroup>
    <UseWindowsForms>true</UseWindowsForms>
  </PropertyGroup>
```

**Comment by abinjaik** [comment]
> The project file needs to have the following by which it reference Windows Forms runtime.
> 
> ```
>   <PropertyGroup>
>     <UseWindowsForms>true</UseWindowsForms>
>   </PropertyGroup>
> ```

@RussKie   Yes, I have this on the .csproj file. It works on my machine. But I am adding this to a Docker Container based out of Linux, its giving me the above error. Seems like WinForms wont work with Linux Containers, Is it?

**Comment by RussKie** [comment]
> WinForms wont work with Linux Containers, Is it?

No.

**Comment by abinjaik** [comment]
> > WinForms wont work with Linux Containers, Is it?
> 
> No.

Thanks @RussKie for the reply. Does the ChartWin project on this repo build without having <UseWindowsForms>true</UseWindowsForms> tag?

**Comment by RussKie** [comment]
No
